data/gegno/gegno intro missions.txt

```diff
@@ -0,0 +1,328 @@
+			`	You walk with the Quarg, occasionally having trouble keeping pace with their silent yet brisk steps, as you head towards an area that appears devoid of the other aliens. Eventually, you are led to a smaller room with two more Quarg in waiting, silently sitting at the end of it. They talk, but this time you can hear a few familiar sounding terms, one being "Humani". Despite speaking to each other for several minutes, the Quarg still look at each other in what you assume is something like bewilderment.`
+			`	One of the Quarg waiting in the room finally turns towards you and motions you to take a seat. Their look of confusion disappears when they turn their gaze back upon you. Meanwhile, the other Quarg that was seated has left the room, along with the ones that first brought you here.`
+			choice
+				`	"Who are these aliens I've been seeing?"`
+				`	"Is it a problem that I'm here?"`
+				`	(Stay silent.)`
+			`	The Quarg continues to look at you and bows its head slightly in an apologetic manner, as it seems neither they, nor any other Quarg on this ringworld, can speak your language. You continue to patiently wait in the uneasy silence, withholding your curiosities as almost half an hour goes by.`
+			`	Finally, the silence is broken when the second Quarg from earlier enters the room again. This time, they glare at you much more energetically, and you begin to wonder if they are going to escort you off of the ring.`
+			`	You preemptively begin to stand up, but you are cut short when you see the second Quarg raise its arms outward in a gesture you never thought you'd see from a Quarg. "This must be for you an exhilarating day, young traverser. I strongly imagine this is a first for both of us." It is now obvious that this Quarg is not the same one from earlier, and you begin to see a few differences in their attire and figure. The upbeat rhythm to their speech is not something you would expect of a Quarg.`
+			choice
+				`	"Where am I?"`
+			`	The Quarg sits down across from you, still gesturing their arms about, albeit slowly, in a strange matter. "You have found yourself in the dwelling of the mighty Gegno, who have rooted their hardy empire here in the great galactic east." You wonder if the Quarg's use of boastful adjectives is because of gaps in his translations, or if they are intentionally speaking of the Gegno in that way. Their voice still comes from the lower tones of Quarg speech, clashing with the excited pace they are speaking at.`
+			choice
+				`	"The Gegno?"`
+					goto gegno
+				`	"Are the ships I’ve seen these Gegno?"`
```

Comment:
Idk why exactly but this feels weird to me. Maybe better like this?
```suggestion
				`	"Do the ships I’ve seen belong to these Gegno?"`
```
or smth of the like? Idk mb its just because I'm bad at english

Reply:
I believe I tried something like this before but Anarchist didn't like it, but I'll commit this anyway and we'll see what Derpy and others have to say about it soon.